capstone-frontend/src/components/BookListAddDropdown.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { DropdownButton, Dropdown } from 'react-bootstrap';
+import CreateList from './CreateList'
+import '../styles/Book.css';
+
+const addBookToBookList = async (bookId, bookListJson) => {
+
+  const bookListUpdateJson = {
+    "id": bookListJson.id,
+    "add_gbookIDs": bookId,
+  }
+
+  // Update BookList in Firebase
+  fetch("/api/booklist", {
+    method: "PUT",
+    body: JSON.stringify(bookListUpdateJson)
+  });
+}
+
+const BookListAddDropdown = ({ book, bookLists, updateBookLists }) => {
+  if (bookLists.length > 0) {
+    return (
+      <DropdownButton id="button-list-add" className="dropdown-add" title="Add to List">
+        {
+          bookLists.map(bookList =>
+            <Dropdown.Item key={bookList.id}
+              onSelect={() => addBookToBookList(book.id, bookList.id)}>
```

Comment:
Similar to a comment I made on Steven's last PR, I think it would be a good idea to update the servlet responsible for adding books to a book list to be able to serve a "batch request." 

The servlet could take in a list of book IDs and a book list ID and add all the books to the book list at once, so you would only need to make one `fetch` call here in the frontend.

Reply:
In the case of this button, the button is only displayed on a button for a specific book so there should be no need for batch request on this button, however, I do agree it would be nice to have a feature like that for if we want some sort of checkbox or "add all books" kind of system!

We will have to discuss how much work it would be to implement.